### PR TITLE
BETA: Register xgamerfreakz.is-a.dev

### PR DESCRIPTION
**Pull request by XgfZ**
Added `xgamerfreakz.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

**Review by triach-rold — CHANGES_REQUESTED** [review]
Website is unreachable and the repo provided is empty. The changes in the other two files (xgfz.json/xgamerfreakz.json) depend on the approval of the first one.

**Review by XgfZ — COMMENTED** [review]
Thanks

File: domains/xgfz.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "XgfZ",
+        "email": "mytab01910@gmail.com"
+    },
+    "record": {
+        "URL": "xgfz.is-a.dev"
```
Comment (21Z) [inline domains/xgfz.json:7]: `URL` must contain protocol.
You cannot redirect the domain to itself.

File: domains/xgamerfreakz.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "XgfZ",
+        "email": "mytab01910@gmail.com"
+    },
+    "record": {
+        "URL": "xgamerfreakz.is-a.dev"
```
Comment (21Z) [inline domains/xgamerfreakz.json:7]: `URL` must contain protocol
You cannot redirect the website to itself.

File: domains/xgamerfreakz.json
Comment (21Z) [inline domains/xgamerfreakz.json:1]: Duplicate domain (domains are always in small letters)

**Review by 21Z — CHANGES_REQUESTED** [review]

**Pull request closed without merging**